src/FixerConfiguration/FixerConfigurationResolver.php

```diff
@@ -116,4 +113,25 @@ public function resolve(array $options)
 
         return $resolver->resolve($options);
     }
+
+    /**
+     * @param FixerOptionInterface $option
+     *
+     * @throws \LogicException when the option is already defined
+     *
+     * @return $this
+     */
+    private function addOption(FixerOptionInterface $option)
```

Comment:
I prefer to have this method helper, so it automatically check `FixerOptionInterace` type on syntax level.